scripts/build.js

```diff
@@ -1,115 +1,119 @@
 // Do this as the first thing so that any code reading it knows the right env.
-process.env.NODE_ENV = 'production';
-
-var chalk = require('chalk');
-var fs = require('fs');
-var path = require('path');
-var filesize = require('filesize');
-var gzipSize = require('gzip-size').sync;
-var rimrafSync = require('rimraf').sync;
-var webpack = require('webpack');
-var config = require('../config/webpack.config.prod');
-var paths = require('../config/paths');
-var checkRequiredFiles = require('./utils/checkRequiredFiles');
-var recursive = require('recursive-readdir');
-var stripAnsi = require('strip-ansi');
-var formatMessage = require('./utils/utils').formatMessage;
-var isLikelyASyntaxError = require('./utils/utils').isLikelyASyntaxError;
-
-checkRequiredFiles();
+import chalk from 'chalk'
+import fs from 'fs'
+import path from 'path'
+import filesize from 'filesize'
+import gzipSize from 'gzip-size'
+import rimraf from 'rimraf'
+import webpack from 'webpack'
+import config from '../config/webpack.config.prod.js'
+import { appBuild, appPackageJson } from '../config/paths.js'
+import checkRequiredFiles from './utils/checkRequiredFiles.js'
+import recursive from 'recursive-readdir'
+import stripAnsi from 'strip-ansi'
+import { formatMessage, isLikelyASyntaxError } from './utils/utils.js'
+import { createRequire } from 'module'
+const require = createRequire(import.meta.url)
+
+process.env.NODE_ENV = 'production'
+
+checkRequiredFiles()
 
 // Input: /User/dan/app/build/static/js/main.82be8.js
 // Output: /static/js/main.js
-function removeFileNameHash(fileName) {
+function removeFileNameHash (fileName) {
   return fileName
-    .replace(paths.appBuild, '')
-    .replace(/\/?(.*)(\.\w+)(\.js|\.css)/, (match, p1, p2, p3) => p1 + p3);
+    .replace(appBuild, '')
```

Comment:
I like `paths.appBuild` better than `appBuild` so I don't need to remember what `appBuild` (or other vars that come from path) references.